### ISSUE TITLE
Update: Two-panel browser

### ISSUE THREAD

**Issue by scripting** [scripting/blue.feedland#7]
This is the URL.

http://scripting.com/code/blueskyreader/?subs=davew

And a screen shot..

![image](https://github.com/scripting/blue.feedland/assets/1686843/3dc790fd-7253-4627-b809-9a7d6473f07b)

I'm continuing to work on it.

**Comment by scripting** [comment]
I added a section at the top of the page, with info pulled from the &lt;head> section of the outline.

Also included the outline URL because I'm not sure I understand how the URL is constructed. Does it default to bsky.social? It seems to be able to find the outline even if I just say _davew_ but it can also find outlines for people who use their own domains. 

![image](https://github.com/scripting/blue.feedland/assets/1686843/e39dcf8c-9cd4-4615-a23b-93933687011b)

**Comment by johnspurlock** [comment]
Here is the current logic for that user handle input normalization:

<img width="577" alt="image" src="https://github.com/scripting/blue.feedland/assets/528937/b6a44c9e-4c48-4880-b3f1-3f1aaa8cc1f3">

**Comment by scripting** [comment]
thanks. i figured that's what you're doing. ;-)

btw your code is easy to read. 

most people's code -- not so much.

**Comment by scripting** [comment]
what i want more than anything right now is a way to get a user's profile info.

what info do we have available?

**Comment by scripting** [comment]
btw, the source:markdown element really contains markdown! I was surprised.

do you convert their internal format to markdown?

**Comment by scripting** [comment]
BTW, I just tried something that worked.

https://drummer.land/?url=https://raw.githubusercontent.com/scripting/blue.feedland/main/blueskyreader/source.opml

That's the source for blueskyreader as of last night being viewed as i view it when i edit, in the outliner (drummer).

**Comment by johnspurlock** [comment]
> do you convert their internal format to markdown?

Yes, their Post records are just plain text with "facets" (text spans with start/end offsets into the text with either Link or Mention information), which can be represented as Markdown without losing anything.

**Comment by scripting** [comment]
btw, just noticed that the feedlist looks a million percent better in feedland. you changed the title of the feeds? good move putting the user's name first. 

<img width="814" alt="image" src="https://github.com/scripting/blue.feedland/assets/1686843/4e6449a4-6179-435d-a69d-128759eacaa8">

**Comment by scripting** [comment]
This is what the prototype looks like now.

http://scripting.com/code/blueskyreader/?subs=davew.bsky.social

<img width="926" alt="image" src="https://github.com/scripting/blue.feedland/assets/1686843/c667428f-634d-4a16-ae59-6ba2ae4dbd06">

**Comment by scripting** [comment]
BTW, this is what the equivalent timeline in FeedLand looks like. 

https://blue.feedland.org/?river=true&screenname=dave

<img width="728" alt="image" src="https://github.com/scripting/blue.feedland/assets/1686843/6dca62f4-abe9-46f8-85e1-7959ae78c33a">

The diff betw this and the prototype is that this is not a prototype. ;-)

**Comment by johnspurlock** [comment]
Nice!  I am now pointing to bsky.app proper for the links, instead of staging.bsky.app, now that the Bluesky team enabled it

**Comment by scripting** [comment]
A nice benefit of having souce:markdown is if I accidentally write something that makes sense in markdown, i get the benefit.

<img width="848" alt="image" src="https://github.com/scripting/blue.feedland/assets/1686843/a3f7c188-b967-43e0-9a6e-5401012dd11a">

**Comment by scripting** [comment]
Another comparison for our discussion tomorrow. This is the _All_ tab in my feedland.org news page.

<img width="755" alt="image" src="https://github.com/scripting/blue.feedland/assets/1686843/64a5797f-6953-4aba-9386-6526f2eb58bd">

**Comment by scripting** [comment]
BTW, the philosophy of FeedLand is that everything but your settings and password are publicly accessible.

So you can see my News page.

https://feedland.org/?river=true&screenname=davewiner&catname=all

**Comment by scripting** [comment]
Another thing I'm thinking for for Bluesky and Mastodon.

In 2014 I made a product for Twitter called Happy Friends at happyfriends.camp. It worked all the way until April. Screen shot below.

It's a mailbox style reader for twitter. Already Twitter was too well established for it to gain much traction, but it might work now for these two systems, esp since they have incompatible APIs.

I'm looking for a first thing to put into the market that might have a chance of gaining some users, so we can build on that.

BTW, I also want to do the productizing for the RSS and OPML feeds you have produced. I think that's a very important piece, a foundation for a lot of software. Might not be necessary to support APIs beyond those two formats. 


![image](https://github.com/scripting/blue.feedland/assets/1686843/662206a3-8f2a-4981-a006-d008b65fa772)

**Comment by scripting** [comment]
I just changed the name on my account to scripting.com by creating the TXT record, and it worked, and I was able to get my OPML file.

<img width="722" alt="image" src="https://github.com/scripting/blue.feedland/assets/1686843/2c639ee7-4c51-4dd6-8892-059d581fdb33">

**Comment by scripting** [comment]
Good morning! I'm going to keep dropping notes into this thread.

Note that the display now includes pointers to the opml and rss data. Makes it easy to review. I kept getting confused as to where these things were. 

Most urgent -- get info about the author into the feed. I'm going to make a proposal for how it should look.

![image](https://github.com/scripting/blue.feedland/assets/1686843/5a070025-fb38-457f-a205-8f810763191c)

**Comment by scripting** [comment]
BTW, for reference, I have an app called tweetfeed.org that generates feeds for tweet streams.

http://tweetfeed.org/davewiner/rss.xml

For some reason it's still running. 

A bunch of friends use it, but it never caught on, like everything else.

**Comment by scripting** [comment]
The docs..

http://docs.tweetfeed.org/

I was reminded earlier today that it folds threads. I am happy to share the code for this. It also supports source:markdown for titles. I think tweets should have titles as should posts on Bluesky. The convention would be if the first character in the post is a # then the remaining text on the line is the title of the post. That would be reflected in the RSS. That way feed readers which kind of expect titles (thanks to Google Reader) will do something nice with the post. 

These little things will add up to textcasting. It's going to be a slog made up of lots of little affordances.
